example/gardener-local/kind/cluster/values.yaml

```
@@ -13,6 +13,7 @@ gardener:
       listenAddressZone0: 127.0.0.10
       listenAddressZone1: 127.0.0.11
       listenAddressZone2: 127.0.0.12
+  templateRoot: "."
```

```suggestion
  repositoryRoot: "."
```

Nit: Since for `templateRoot` I'd rather expect something like `./charts`

---

Completely agree, thanks for the input. Changes made 👍